msal/application.py

```diff
@@ -413,7 +413,7 @@ def acquire_token_silent(
         #     verify=self.verify, proxies=self.proxies, timeout=self.timeout,
         #     ) if authority else self.authority
         result = self._acquire_token_silent_from_cache_and_possibly_refresh_it(
-            scopes, account, self.authority, **kwargs)
+            scopes, account, self.authority, force_refresh, **kwargs)
```

Comment:
Good catch! Would you mind change this addition to use the keyword notation? `force_refresh=force_refresh`, as we did at [this line](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/113/files#diff-f7ae931bcdab4b14f3a256baf7d6922aR425)? That way it won't break if we add more new positional parameters later.